core/src/commonMain/kotlin/State.kt

```diff
@@ -104,7 +104,42 @@ public sealed class State {
             public object EncryptionTimedOut : Status()
 
             /** Catch-all for any statuses that are unknown for a platform. */
-            public data class Unknown(val status: Int) : Status()
+            public data class Unknown(val status: Int) : Status() {
+                override fun toString(): String {
+                    return status.toString()
+                }
+            }
```

Comment:
Please revert this change as I think it would make sense for consumers to be able to print an `Unknown` instance and have it take the form of `"Unknown(status=#)"`.

See comment below on `Disconnected.toString` where it handles the abbreviated `Unknown` printout.